Model/Ui/CcConfigProvider.php

```diff
@@ -37,10 +47,40 @@ public function getConfig()
                     'years'  => [OmiseCcConfig::CODE => $this->magentoCcConfig->getCcYears()],
                 ],
                 OmiseCcConfig::CODE => [
-                    'publicKey'      => $this->omiseCcConfig->getPublicKey(),
-                    'offsitePayment' => $this->omiseCcConfig->is3DSecureEnabled()
+                    'publicKey'          => $this->omiseCcConfig->getPublicKey(),
+                    'offsitePayment'     => $this->omiseCcConfig->is3DSecureEnabled(),
+                    'isCustomerLoggedIn' => $this->customer->isLoggedIn(),
+                    'cards'              => $this->getCards(),
                 ],
             ]
         ];
     }
+
+    /**
+     * @return  array
+     */
+    public function getCards()
+    {
+        if (! $this->customer->getMagentoCustomerId() || ! $this->customer->getId()) {
+            return [];
+        }
+
+        $cards = $this->customer->cards(array('order' => 'reverse_chronological'));
```

Comment:
Mixing array syntaxes - can we stick with one of the other? Short probably better

Reply:
Actually according to Magento code style guide, we need to update to `array()` one.
nice catch 👍